project1/styles.css

```
@@ -0,0 +1,86 @@
+blockquote p {
+    font-family: 'Satisfy', cursive;
+    font-size: 30px;
+    line-height: 1;
+    margin-left: 150px;
+    margin-right: 150px;
+    text-align: center;
+}
+
+body {
+
+    color: white;
+}
+
+div {
+    font-size: 40px;
```

In addition to tweaking the `font-size` of your bullets, you may want to also override the browser default for `line-height` to tweak your spacing issues:
```suggestion
    font-size: 40px;
    line-height: 1;
```